src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -144,20 +144,67 @@ class ACCStandaloneDirective(ACCDirective, StandaloneDirective,
 
 
 class ACCRoutineDirective(ACCStandaloneDirective):
-    ''' Class representing a "!$ACC routine" OpenACC directive in PSyIR. '''
+    '''
+    Class representing an "ACC routine" OpenACC directive in PSyIR.
+
+    :param str parallelism: the level of parallelism in the routine, one of
+        "gang", "worker", "vector", "seq". Defaults to "seq".
+
+    '''
+    SUPPORTED_PARALLELISM = ["seq", "vector", "worker", "gang"]
+
+    def __init__(self, parallelism=None, **kwargs):
```

Comment:
Any reason not to use `parallelism="seq"` here for the default value?
- We could then remove the first if condition of the parallelism.setter
- the documented type will match without needing the Optional[]

Reply:
Oh yes, done.